_sass/uno.scss

```diff
@@ -1190,27 +1193,33 @@ hr {
 	vertical-align: middle; }
 
 .panel-title {
-	margin: 0 0 5px 0;
+	margin: 0 0 0 0;
 	font-size: 2.5em;
-	letter-spacing: 4px;
+	font-weight: bold;
+	text-shadow: 0px 3px 0px #0000001f;
+	letter-spacing: 1px;
 	color: #FFF; }
 
 .panel-subtitle {
-	font-family: "Roboto Slab", serif;
+	font-family: "Nunito Sans", serif;
 	font-size: 1.2em;
 	font-weight: lighter;
 	letter-spacing: 3px;
 	color: #cccccc;
 	-webkit-font-smoothing: antialiased; }
 
+.crown-logo {
+	width: 100px;
```

Comment:
Can we make this percentage based?

Reply:
I'm not sure that is really correct, the title text below isn't based on window size and the logo size is chosen relative to that, I'm not sure it should be done on percentage?

Reply:
I'm not sure that is really correct, the title text below isn't based on window size and the logo size is chosen relative to that, I'm not sure it should be done on percentage?